### DISCUSSION TITLE
Using LPS Node firmware with ESP32 module combined with DWM1000 for TDoA

### DISCUSSION THREAD

**Discussion by anthony211212** [bitcraze/discussions#1075] [Q&A]
Has anyone tried this code with the dwm1000 module in esp32? I'm thinking of using it for TDoA but I couldn't find a complete result. does anyone have any suggestions? There are files such as lpsTdoaTag.c, uwb_tdoa_anchor2.c and uwb_tdoa_anchor3.c in the files. but I couldn't understand exactly what they are. I would appreciate if anyone can guide me. My goal is to use multiple tags and anchors with TDoA method. I am thinking of using esp32 for this. Thanks for your help in advance.

**Comment by knmcguire** [comment]
Duplicate of https://github.com/orgs/bitcraze/discussions/1072.

**Reply by anthony211212** [reply]
Hello. Sorry for duplicate. I can not see the other page therefore I asked again.

**Reply by anthony211212** [reply]
Why the topic is closed? (#1072)

**Reply by knmcguire** [reply]
I only closed the issue; https://github.com/bitcraze/lps-node-firmware/issues/63

. https://github.com/orgs/bitcraze/discussions/1072 is still open.
